svelte-web-frontend/src/lib/model/settings.ts

```diff
@@ -1,14 +1,26 @@
 import { dev } from '$app/environment';
 import { writable } from 'svelte/store';
 
-export const debugMode = writable(dev);
```

Comment:
with this persistance change, I think we can default debugMode to `false` and remove the references to `dev` and the import from `$app/environment`.